src/pages/Login.tsx

```diff
@@ -0,0 +1,73 @@
+import NavigationBar from '@/components/NavigationBar';
+import styled from '@emotion/styled';
+import { useNavigate } from 'react-router-dom';
+
+const StyledLoginComponentContainerDiv = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+const StyledLoginComponentDiv = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+  width: 100vh;
+`;
+
+const StyledKakoLogo = styled.div`
+  ${({ theme }) => theme.typography.title2Bold}
+  margin: ${({ theme }) => theme.spacing.spacing11};
+  font-size: 30px;
+`;
+
+const StyledInput = styled.input`
+  border: none;
+  border-style: solid;
+  border-color: ${({ theme }) => theme.palette.gray400};
+  border-width: 0px 0px 1px;
+  padding: ${({ theme }) => theme.spacing.spacing4};
+  width: 52%;
+  min-height: 20px;
+`;
+
+const StyledButton = styled.button`
+  margin-top: ${({ theme }) => theme.spacing.spacing12};
+  background-color: ${({ theme }) => theme.sementicPalette.kakaoYellow};
+  border: 1px solid ${({ theme }) => theme.sementicPalette.kakaoYellow};
+  border-radius: 4px;
+  width: 52%;
+  height: 40px;
+  a {
+    text-decoration: none;
+  }
+`;
+
+const Login = () => {
+  const navigate = useNavigate();
+
+  const handelBackOrHome = () => {
+    if (window.history.length > 2) {
+      navigate(-1);
+    } else {
+      navigate('/');
+    }
+  };
+  return (
+    <>
+      <NavigationBar />
+      <StyledLoginComponentContainerDiv>
+        <StyledLoginComponentDiv>
+          <StyledKakoLogo>kakao</StyledKakoLogo>
+          <StyledInput id='loginid' placeholder='이메일'></StyledInput>
+          <StyledInput id='passwd' placeholder='비밀번호'></StyledInput>
```

Comment:
여기두 self-closing tag 사용하시면 좋을 것 같아용!

```suggestion
          <StyledInput id='loginid' placeholder='이메일' />
          <StyledInput id='passwd' placeholder='비밀번호' />
```

Reply:
그리고 input에는 각 의도에 맞는 type이 잘 들어가있으면 좋아용!
지금은 비밀번호이지만 이렇게 입력하면 비밀번호가 다 보이는 형태라서,
input type="email", input type="password" 를 활용해서 타입을 넣어주시면 좋을 것 같네용 👍 
<img width="477" alt="image" src="https://github.com/user-attachments/assets/f79258a0-cdbb-4e16-bb0c-2cedf75a6fc0" />